grass7/imagery/i.sentinel/i.sentinel.mask/i.sentinel.mask.py

```diff
@@ -118,6 +118,12 @@
 #% required : no
 #% guisection: Metadata
 #%end
+#%option G_OPT_F_INPUT
+#% key: metadata
+#% description: name of Sentinel metadata json dump
```

Comment:
```suggestion
#% description: Name of Sentinel metadata json dump
```

see also comment above for predefined path